### DISCUSSION TITLE
Increasing LWCF without major impact on SWCF

### DISCUSSION THREAD

**Discussion by JulioTBacmeister** [NCAR/amwg_dev#247] [TF2: Tuning]
In many respects simulations with DCS=250 microns (instead of current default of 500) look good. However, global LWCF is reduced to value below 20 Wm-2 from a value around 23 Wm-2 in default. Current CERES-EBAF estimates are closer to 25 Wm-2.

This discussion is meant to collect ideas on how to target LWCF in the model without significant imapct on SWCF.

**Comment by JulioTBacmeister** [comment]
PPE results from Trude Eidhammer showing impacts of parameters

![image](https://user-images.githubusercontent.com/9221710/232120263-13312b54-2804-43b8-b692-a46010e01df1.png)

**Reply by adamrher** [reply]
@andrewgettelman @Katetc

**Reply by tilmes** [reply]
We could try increasing the dust tuning/scaling factor for ice clouds which would increase the ice number:
So for example change hetfrz_dust_scalfac=0.05 to hetfrz_dust_scalfac 0.1 or just as a test to 0.5 and see what happens.

**Reply by zarzycki** [reply]
From Trude's results, the two parameters that immediately stand out to me are `micro_mg_vtrmi_factor` and `zmconv_c0_lnd` as having some contribution to LWCF sensitivity.

The former seems to be a big knob but the SWCF and LWCF sensitivities oppose vs. having the same sign for `micro_mg_dcs`. Makes sense the ice fall velocity would impact LWCF.

The latter could help draw back LWCF although it would seem to also have marginal impacts on other parameters. I know others have looked at `zmconv_c0_lnd` being tweaked, so perhaps they have insight?

**Reply by tilmes** [reply]
Trude did not include this factor in the table: "hetfrz_dust_scalfac' is there a way to do that?

**Reply by Katetc** [reply]
@tilmes Unfortunately, "hetfrz_dust_scalfac" was not included as a varied factor in this PPE. We can add it to the list for a future survey though.

**Comment by Katetc** [comment]
Here is a zoomed in copy of the "global" section of that figure (but as it is oriented in the current paper draft):
![Fig11PPEglobal](https://user-images.githubusercontent.com/13952101/232127526-76f47758-7e2f-4a89-85c1-d25356b3ff3b.jpg)

I think if you are looking for parameters that have a impact on LWCF but less of an impact on SWCF there are a few options:
microp_aero_wsubi_scale and micro_mg_vtrmi_factor look like the two most promising ones to me. As Colin said, micro_mg_vtrmi_factor is the ice fall speed velocity so reducing that might help ice stick around in the upper levels longer, and I'm not super familiar with microp_aero_wsubi_scale, but the description in the paper is "Subgrid velocity for ice activation scaling" so that parameter is ice specific as well.

**Reply by adamrher** [reply]
I like the idea of reducing the fall speeds of ice hydrometeors ... Jiang also suggested this in todays mtg. Am I reading the plot correctly, that by increasing ```micro_mg_vtrmi_factor``` it will decrease LWCF? Suggesting that a smaller value will reduce fall velocities and increase LWCF? The namelist defintion is:

```
<entry id="micro_mg_vtrmi_factor" type="real" category="microphys"
       group="micro_mg_nl" valid_values="" >
Unitless scaling factor for ice fall speed to account for sub-grid scale ice crystal shape variability.
Default: 1.0
</entry>
```

So the process we are tuning here is ice crystal shape variability ... similar to Julio's "habit" suggestion. The value is currently set to 1.0. How much should we reduce it to target the 5 W/m2 LWCF bias we currently have?

**Reply by bstephens82** [reply]
I tried halving this fall speed parameter to 0.5 with the 099 tag, and it brought LWCF up by about 0.3 W/m2 to 24.8 W/m2 or so.  It brought SWCF down in magnitude from -50 W/m2 to around -48.8 W/m2.  I'm not sure how extreme halving this parameter is, maybe @andrewgettelman would have an opinion on that.  I am trying a test with vtrmi_factor=0.25 now.

Also testing the parameter micro_mg_autocon_lwp_exp to see what increasing that will do.  I tried reducing it to 2.0 (from 2.47), but that was too much.  It brought SWCF to ~-40, killing the marine Sc, and LWCF up to ~30 W/m2.  I'll try 2.25.  Maybe some combination of changes to these parameters and dcs can get us to some decent results.

**Reply by bstephens82** [reply]
Also @Katetc am I missing something, or are the results in your figure different from the ones in Julio's?  For example, the parameter micro_mg_autocon_lwp_exp in Julio's figure seems to have a large impact on SWCF in the global section (bright red) but very little effect on LWCF (almost white).  But in your figure, micro_mg_autocon_lwp_exp is blue for SWCF and slightly red for LWCF.

**Comment by andrewgettelman** [comment]
It's rough, but I think an uncertainty in fall speed of a factor of 2 (0.5 to 2 on the scaling) is probably well within the uncertainty range. I think there were some experiments at 0.1 to 10 (factor of 10) and that seems a bit harder to justify. Maybe changing the detrained ice size back to a small value (say 2-10 microns, I think it's 61 now but used to be 1 micron) might have similar impacts (lots more small ice so more mass).....

I can try to look at it a bit more this week.

**Reply by bstephens82** [reply]
> It's rough, but I think an uncertainty in fall speed of a factor of 2 (0.5 to 2 on the scaling) is probably well within the uncertainty range. I think there were some experiments at 0.1 to 10 (factor of 10) and that seems a bit harder to justify. Maybe changing the detrained ice size back to a small value (say 2-10 microns, I think it's 61 now but used to be 1 micron) might have similar impacts (lots more small ice so more mass).....
> 
> I can try to look at it a bit more this week.

Decreasing clubb_detice_rad down to 10 microns (ie clubb_detice_rad=0.00001) did bring up the global mean LWCF (a lot, to over 30 W/m2), but it also increased the magnitude of SWCF by about the same amount, so it went "up" to about -56 W/m2.  So this seems to affect both about the same in the global mean.

**Comment by JulioTBacmeister** [comment]
@andrewgettelman Thanks for this suggestion.  I think it would be worth trying this.  One question: To what extent would decreasing detrained ice size simply counteract the impact of decreasing DCS?

**Comment by spraghuraman** [comment]
@JulioTBacmeister, what is the simulated clear-sky OLR? As LWCRE = Clear-sky OLR - All-sky OLR, it could also be that the All-sky OLR is fine and clear-sky OLR needs tuning. Ways to increase clear-sky OLR and hence LWCRE: warmer surface or atmospheric temperatures or less water vapor. Ways to decrease all-sky OLR and hence increase LWCRE without affecting SW: decreasing cloud top temperatures.

**Reply by bstephens82** [reply]
Thanks @shivpriyamr for your advice.  When you say decreasing cloud top temperatures, do you know a simple way of doing that?

**Reply by adamrher** [reply]
I can't think of any other way than to raise the detrainment level in ZM. This could be related to why tiedke_add increases LWCF in the PPE; it may be raising the level of neutral buoyancy and therefore the cloud tops.

**Reply by spraghuraman** [reply]
@bstephens82 not sure of a simple way but @adamrher's is a good idea. It already gets the peak high cloud coverage/detrainment correctly (although it vastly overestimates the amount; figure attached) so I do worry if we push it up higher it may cause other biases.
<img width="372" alt="image" src="https://user-images.githubusercontent.com/131018651/232622847-4f57c077-627f-4532-a6ad-7d4db3abe3d2.png">

**Reply by adamrher** [reply]
I tend to agree with @shivpriyamr. The PPE shows tiedke_add increases cloud fraction quite a bit too, so it probably would just make matters worse, according to his plot.

I also don't know that tiedke_add would actually raise the neutral buoyancy level; it may just make CAPE and the cloud base mass fluxes systemically larger, and cause it just detrain more at the same level.

**Reply by swrneale** [reply]
@shivpriyamr can you separate out he CLUBB versus ZM components to the CAM cloud profile here so we know the relative contributions?

**Reply by bstephens82** [reply]
> I tend to agree with @shivpriyamr. The PPE shows tiedke_add increases cloud fraction quite a bit too, so it probably would just make matters worse, according to his plot.
> 
> I also don't know that tiedke_add would actually raise the neutral buoyancy level; it may just make CAPE and the cloud base mass fluxes systemically larger, and cause it just detrain more at the same level.

Hmm ok.  I think by now I've found a couple of different sets of parameters that will give decent SWCF and LWCF in the latest tags, but always at the cost of a RESTOM that is too large (typically well over 3.0).  I am guessing that this comes about as an indirect effect of bringing down SWCF but trying to hold LWCF in the mid-20 W/m2 range.  Interestingly there seems not to be much difference between the clear-sky TOA SW or LW flux between runs that show RESTOM~1 and runs that show larger RESTOM.  In particular the LW TOA clear-sky flux only differs by hundredths of a W/m2, and the SW TOA clear-sky flux differs by tenths of a W/m2.  So I'm not sure how to interpret that exactly, unless small differences like that can significantly impact RESTOM.  It does look like there is more water vapor in the upper atmosphere in runs that show a larger RESTOM, which I suppose would block more outgoing longwave.  If that is what is going on, is there a way to tweak that?  

- This is from the run I did with the 099 tag with micro_mg_dcs=250, where the LWCF is very low (around 19 W/m2) but RESTOM is about 0.9:
![set4_ANN_SHUM_MERRA_obsc](https://user-images.githubusercontent.com/59519056/232820383-6eeb7439-d16d-42c2-b9fc-4e402cb6902e.png)

- This is from a new run I did where I leave dcs alone but use vtrmi_factor=0.5 and micro_mg_autocon_lwp_exp=2.3.  This run has LWCF=25.50, SWCF=-46.14, and RESTOM=6.224.
![set4_ANN_SHUM_MERRA_obsc](https://user-images.githubusercontent.com/59519056/232820688-997547d3-78d9-48f7-ae7c-0483a724d9e1.png)

**Reply by adamrher** [reply]
I was looking at the cloud in the 099+dcs=250 run in the table [here](https://user-images.githubusercontent.com/59519056/231841566-cba9275b-7d1f-4335-966c-8dc185937252.png). It seems like this run reduces cloud fraction a lot, and through the whole column. But if I recall correctly, it was the only way to tame the SWCF over the southern ocean, and so something we do want to try to retain. It may make sense to combine this with vtrmi_factor=0.5 and tiedke_add, to boost high cloud fraction and ice content. Maybe you've got a run like this already?

**Reply by bstephens82** [reply]
@adamrher here is an 099-tag run with dcs=250, vtrmi_factor=0.5, and tiedke_add=1.5.  Compared to the simple dcs=250 run, it brings up LWCF slightly to about 20 W/m2, but it also has a large RESTOM over 6.0.  If you think a different value of tiedke_add would be worth checking out let me know.
https://webext.cgd.ucar.edu/F2000climo/dflt099_vtrmf0.5_dcs250_tiedke1.5/atm/dflt099_vtrmf0.5_dcs250_tiedke1.5-obs.0001_1/

Maybe the best overall run I've got so far is this one, with dcs=400, vtrmi_factor=0.5, and autocon_lwp_exp=2.4 (down from a default value of 2.47).  This one has a RESTOM of a little less than 4.0, and the global mean LWCF is a bit low, about 23 W/m2, but I think SWCF looks not too bad:
https://webext.cgd.ucar.edu/F2000climo/dflt099_vtrmf0.5_lwpexp2.4_dcs400/atm/dflt099_vtrmf0.5_lwpexp2.4_dcs400-obs.0001_1/

**Reply by adamrher** [reply]
@bstephens82 thanks for testing tiedke_add. It seems to have a marginal impact. I've got another idea to crank up ZM detrainment and increase high cloud amount --- Thomas' mods to includes the ice phase in the plume ensemble. This serves to increase plume buoyancy above the freeze line. The sourcemods are old; they pertain to zm2 in cam6_3_019. But with a little hacking, I got them to build in run with tag 97. Mind giving them a whirl?

```
/glade/u/home/aherring/src/cam6_3_097/usr_src/thtmods/zm_conv.F90
/glade/u/home/aherring/src/cam6_3_097/usr_src/thtmods/zm_conv_intr.F90
```

I'd prefer dcs=250E-6 in this run. Up to you if you want to throw tiedke_add on top of this too.

**Reply by adamrher** [reply]
yes @bstephens82, please run if you can. no reason not to, IMO.

**Reply by spraghuraman** [reply]
> @shivpriyamr can you separate out he CLUBB versus ZM components to the CAM cloud profile here so we know the relative contributions?

@swrneale I haven't looked at that, I could. I think I downloaded the CAM cloud profile from ESGF (clcalipso_CFmon_CESM2_amip_r1i1p1f1_gn_195001-201412.nc). Could you point me to the /glade directory where there is cl_CLUBB or cl_ZM? Ideally it would be with the CALIPSO simulator turned on ("clcalipso") but no worries if that's not there.

**Comment by JulioTBacmeister** [comment]
@shivpriyamr In fact Ben and I were just looking at clear sky OLR and it indeed looks too low.

**Comment by JulioTBacmeister** [comment]
Summary of radiative fluxes through tag 105 and tag 105+DCS=250

![image](https://user-images.githubusercontent.com/9221710/232822993-5ea1ebed-4a60-4270-b41d-32d6b2477bb1.png)

**Reply by andrewgettelman** [reply]
I think Tag 105 looks decent in this metric. What about just dialing low clouds back from there?

**Reply by adamrher** [reply]
@andrewgettelman I think the reason is the -50 W/m2 / tag 105 has decent Sc decks. And so dialing back low clouds with clubb_c8 or clubb_gamma gives us too weak Sc decks. See [here c8 and clubb_gamma expts](https://github.com/NCAR/amwg_dev/discussions/225#discussioncomment-5419735). (note cloud focing in tag 99 is ~same as tag 105).

**Reply by JulioTBacmeister** [reply]
@andrewgettelman, @adamrher is correct, we'd like to have some way reducing SWCF somewhere *other* than in the subtropical stratus decks

**Comment by JulioTBacmeister** [comment]
Why not increase condensate detrainment from ZM by simply reducing the
convective autoconversion rate in ZM? That is a namelist parameter.

**Reply by adamrher** [reply]
Looking at the PPE, it's not very responsive to the zmconv_c0_lnd/ocn autoconversion factors.

**Reply by bstephens82** [reply]
Yeah, I was looking at that, zmconv_c0_lnd does impact SWCF and LWCF with opposite signs, but might need a larger change to have a noticeable impact?  I started a run trying it just to see.  @adamrher should I still try your suggestion above with the sourcemods?

**Reply by zarzycki** [reply]
In my hindcasts, I've found zmconv_c0_lnd generally has more of a LWCF impact that SWCF impact, although agree w/ Ben that the signs flip for SWCF/LWCF in Trude's plots so seems worth an exploration. Given that, I wonder if tweaking zmconv_c0_lnd could offset the micro_mg_dcs kick... If micro_mg_dcs doesn't come all the way down to 250 (maybe 300 or 350) that would be more tractable, I assume...

**Reply by JulioTBacmeister** [reply]
So, despite the apparent lack of impact in the PPE I still think it would be worth trying some runs with changes to zmconv_c0 ...

**Reply by bstephens82** [reply]
> So, despite the apparent lack of impact in the PPE I still think it would be worth trying some runs with changes to zmconv_c0 ...

I have run a test modifying this, I'll post here later this morning.

**Reply by bstephens82** [reply]
Okay, I was wondering if it might be possible to use zmconv_c0_lnd to impact SWCF and LWCF in opposite ways, but yes it turns out there is just very little sensitivity to this parameter.  Reducing it by 1/3 from 0.0075 to 0.005 has almost no impact at all on either SWCF or LWCF.

**Reply by swrneale** [reply]
@ben actually in the past we have been pretty liberal with changing the values of the c0 parameters. If I recall correctly, I think it was CAM5 to CAM6 where we have changed these values by an order of magnitude.

**Reply by swrneale** [reply]
@ben also, I think I had mentioned this to you before, but I think we should revisit how ZM and CLUBB cloud fraction and in-cloud liquids are combined to form a net emissivity for the purposes of radiation. Checkout  conv_water.F90. This where this happens and @andrewgettelman and I have implemented this in the past. Prior to this, convective cloud cores had been kind of invisible to radiation. Maybe we can get some leverage with this. Out on travel this week. Could discuss next.

**Comment by JulioTBacmeister** [comment]
Yes, You're absolutely correct. That's curious.  Maybe I should reconsider
my suggestion.

**Comment by JulioTBacmeister** [comment]
Hmmm. So I read Adam's comment first.  As far as I can see the PPE in fact
doesn't suggest that zmconv_c0_{lnd,ocn} is an effective way to change LWCF
or SWCF. What are you referring to Ben? Did you do runs with modified
zmconv_c0 at some point?

**Comment by tto061** [comment]
Hi Adam, Julio, and all,

The SourceMods attached here are my modified zm_conv routines updated to 
tag cam 6_3_106. 
[zm_conv_106.tar.gz](https://github.com/NCAR/amwg_dev/files/11274815/zm_conv_106.tar.gz)

For a test I would suggest to try all the (hard-coded) settings as they 
stand in these routines. These match NorESM tunings, and they all try to 
increase the heating level of zm_conv, to some extent successfully (in 
particular monsoons tend to be better).

However, if you want just the ice mods and my other pure "physics" mods, 
you might want to turn to false the three logical parameters 
"second_call", "retrigger", and "use_cin" in the declarations (line 75) 
of zm_conv.F90 (so just "tht_tweaks" is set to .true.).

Best regards
Thomas


Thomas Toniazzo
MISU          Stockholms university
SE-106 91 Stockholm         Sverige

**Reply by bstephens82** [reply]
Hi @tto061 your attachments didn't come through (I think files can only be posted directly on github).  Are they on glade somewhere?

**Reply by tto061** [reply]
Ben, they should be attached here now hopefully

**Comment by bstephens82** [comment]
@Katetc @JulioTBacmeister @zarzycki @adamrher @vlarson Here are some results from a few of my latest runs.  This is a table with various global mean values of interest, but the top row has links to the full output for each run.  

The parameter changes in these runs give relatively decent cloud forcings, but they typically increase RESTOM beyond acceptable values.
  
https://docs.google.com/spreadsheets/d/1fYeWnCmUJAPWKxJD56te0aHpA8azVjMV91wOvBnlvS0/edit?usp=sharing
